src/js/api.js

```diff
@@ -1,21 +1,21 @@
 "use strict";
 
-var rest = require("rest");
-var pathPrefix = require("rest/interceptor/pathPrefix");
-var mime = require("rest/interceptor/mime");
-var errorCode = require("rest/interceptor/errorCode");
-var defaultRequest = require("rest/interceptor/defaultRequest");
-var hateoas = require("rest/interceptor/hateoas");
+import rest from "rest";
+import pathPrefix from "rest/interceptor/pathPrefix";
+import mime from "rest/interceptor/mime";
+import errorCode from "rest/interceptor/errorCode";
+import defaultRequest from "rest/interceptor/defaultRequest";
+import hateoas from "rest/interceptor/hateoas";
```

Comment:
This is used to automatically convert JSON bodies to entities (for now). I wish we'll use hateoas someday anyway :)